### ISSUE TITLE
Replace print statements with python logging

### ISSUE THREAD

**Issue by kdlong** [kdlong/WZAnalysis#7]
Would be much more useful for having configurable verbosity, useful for debugging

**Comment by dteague** [comment]
Is there a way to toggle the debug statements? We might want to put a flag for turning on/off. I like the ones I see so far!

**Comment by kdlong** [comment]
Yes, here: https://github.com/kdlong/VVAnalysis/blob/master/Utilities/python/SelectorTools.py#L14

Could be set by a flag passed as a command line arg
